src/pyscheduling/SMSP/SingleMachine.py

```diff
@@ -968,20 +995,31 @@ def wiCi(self):
         if self.instance != None:
                 self.machine.total_weighted_completion_time(self.instance)
         self.objective_value = self.machine.objective
+        return self.objective_value
 
     def wiTi(self):
         """Sets the job_schedule of the machine and affects the total weighted lateness to the objective_value attribute
         """
         if self.instance != None:
                 self.machine.total_weighted_lateness(self.instance)
         self.objective_value = self.machine.objective
+        return self.objective_value
 
     def Cmax(self):
         """Sets the job_schedule of the machine and affects the makespan to the objective_value attribute
         """
         if self.instance != None:
                 self.machine.completion_time(self.instance)
         self.objective_value = self.machine.objective
+        return self.objective_value
+
+    def Lmax(self):
+        """Sets the job_schedule of the machine and affects the maximum lateness to the objective_value attribute
+        """
+        if self.instance != None:
+                self.machine.maximum_lateness(self.instance)
+        self.objective_value = self.machine.objective
```

Comment:
It is better to refactor machine.objective to machine.objective_value for consistency and leave objective attribute for the type of objective rather than its value. 

We can do this in a later ticket